content/git/concepts/config/config.md

```diff
@@ -40,3 +40,24 @@ To remove a specific configuration setting, use the `--unset` flag with the `git
 ```shell
 git config --global --unset user.email
 ```
+
+## Git Configuration Files
+
+Git stores all of these configuration settings in simple files which can be found in different places:
+
+- Global settings are saved in a file called `.gitconfig` in the user's home directory.
+- Project-specific settings are saved in `.git/config` within the repository.
+
+Settings in the repository's local configuration file will override settings in the global file.
+
+These files are just plain text files and can be edited directly if you'd like, rather than using the `git config` commands. A basic configuration file that sets user info and adds some aliases may look like:
```

Comment:
```suggestion
These files are just plain text files and can be edited directly rather than using the `git config` commands. A basic configuration file that sets user info and adds some aliases may look like:
```

Reply:
docs avoids the use of first and second person pronouns (I you we)